frontend/CHANGELOG.md

```diff
@@ -9,3 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Added `max_messages` to `events` and `events_window` ([#2947](https://github.com/open-ic/open-chat/pull/2947))
+
+### Added
```

Comment:
Nit: Added should come first

Reply:
Oh I thought I saw it the other way round when you were looking at examples